test/utils/rfqm_health_check_test.ts

```diff
@@ -141,7 +141,9 @@ describe('RFQm Health Check', () => {
                 const degradedIssues = issues.filter(({ status }) => status === HealthCheckStatus.Degraded);
 
                 expect(degradedIssues).to.have.length(1);
-                expect(degradedIssues[0].description).to.contain('0x00');
+                expect(degradedIssues[0].description).to.contain(
+                    'Less than two workers have a balance above the degraded threshold (0.1)',
```

Comment:
nit: thoughts on leaving out the `(0.1)`? 

Pros - makes it easier to change in the future
Cons - makes it easier to change in the future